test/unit/testCoApplicant.js

```diff
@@ -1,10 +1,9 @@
-const initSteps = require('app/core/initSteps'),
-    assert = require('chai').assert,
+const initSteps = require('app/core/initSteps');
+const assert = require('chai').assert,
     {isNil} = require('lodash');
```

Comment:
assert variable has been changed to be individually declared as const (was previously declared in tandem with initSteps.  Thus to keep this standardised we should also declare {isNil} individually as a const